src/Tickets/Emails/Email/Completed_Order.php

```diff
@@ -0,0 +1,402 @@
+<?php
+/**
+ * Class Completed_Order
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails\Email;
+
+use \TEC\Tickets\Emails\Email_Template;
+
+/**
+ * Class Completed_Order
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Completed_Order extends \TEC\Tickets\Emails\Email_Abstract {
+
+	/**
+	 * Email ID.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $id = 'tec_tickets_emails_completed_order';
+
+	/**
+	 * Email template.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $template = 'admin-new-order';
+
+	/**
+	 * Email recipient.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $recipient = 'admin';
+
+	/**
+	 * Enabled option key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_enabled = 'tec-tickets-emails-completed-order-enabled';
+
+	/**
+	 * Recipient option key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_recipient = 'tec-tickets-emails-completed-order-recipient';
+
+	/**
+	 * Subject option key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_subject = 'tec-tickets-emails-completed-order-subject';
+
+	/**
+	 * Email heading option key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_heading = 'tec-tickets-emails-completed-order-heading';
+
+	/**
+	 * Email heading option key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_add_content = 'tec-tickets-emails-completed-order-add-content';
+
+	/**
+	 * Checks if this email is enabled.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_enabled(): bool {
+		return tribe_is_truthy( tribe_get_option( static::$option_enabled, true ) );
+	}
+
+	/**
+	 * Checks if this email is sent to customer.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_customer_email(): bool {
+		return false;
+	}
+
+	/**
+	 * Get email title.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email title.
+	 */
+	public function get_title(): string {
+		return esc_html__( 'Completed Order', 'event-tickets' );
+	}
+
+	/**
+	 * Get email recipient.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email recipient.
+	 */
+	public function get_recipient(): string {
+		$recipient = tribe_get_option( static::$option_recipient, $this->get_default_recipient() );
+
+		/**
+		 * Allow filtering the email recipient for Completed Order.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $heading  The email recipient.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$recipient = apply_filters( 'tec_tickets_emails_completed_order_recipient', $recipient, self::$id, $this->template );
```

Comment:
I was thinking about the actual name of the hook if we were to actually use the id name. It would be `tec_tickets_emails_recipient_tec_tickets_emails_completed_order` vs just using `tec_tickets_emails_completed_order_recipient`. Plus, I didn't see many examples in our codebase where we use variables within a hook name. What are your thoughts about using the `template` name instead of `id`? Example: `tec_tickets_emails_{$template}_recipient`. That would make more sense as far as readability. And each will have to be a unique template name anyway.